### PR TITLE
Add OpenRRP schedule

### LINKED ISSUE #147: Add OpenRRP schedule
This issue tracks updating the schedule for May 14-15th 2024's OpenRRP workshop.

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #147 

This PR adds an internal schedule. Links are already there for day 1, but we'll need to add for day 2 as well (#165).

Let me know if times make sense to you all, including food times!

### CODE REVIEW

File: docs/workshop-schedule.md [hunk 1]
```diff
@@ -3,39 +3,49 @@ title: Workshop Schedule
...
+| 2:45 - 3:00 pm      | _Coffee break_                                                                                                                                                |
+| 3:00 - 3:45 pm      | [Organizing code in scripts and notebooks](https://docs.google.com/presentation/d/1AJr6uQhwLnZfis1wNc_e2XY4XSMEuVscIfAsVgnM5Bk/edit?usp=sharing)              |
+| 3:45 - 4:15 pm      | [Managing packages and environments](https://docs.google.com/presentation/d/1GCbu2F6LeEPOu5DzDsTgwu1__9YDVydvPo911fBG1i0/edit?usp=sharing)                    |
+| 4:15 - 4:45 pm      | [Working with branches in Git](https://docs.google.com/presentation/d/1s7BSHgTSDuXIzI1ROS-JSneB6NXfQVWOec6lhc8eIWA/edit?usp=sharing)                          |
+| 4:45 pm             | _Adjourn_                                                                                                                                                     |
+| 5:30 pm             | _Optional_ Meet in the hotel lobby to Lyft to dinner                                                                                                          |
```
Comment (jashapiro) [inline docs/workshop-schedule.md:33]: Is everyone in the same hotel? Probably not, and some of us will probably drive folks?

Comment (sjspielman) [reply docs/workshop-schedule.md:33]: Maybe just remove this line from the schedule altogether? We're a small enough group that we can just figure this out and communicate it when we adjourn.

File: docs/workshop-schedule.md [hunk 2]
```diff
@@ -3,39 +3,49 @@ title: Workshop Schedule
...
+|                     | [_Resource_: UNIX Cheatsheet (pdf)](resources/unix_quick_reference.pdf)                                                                                       |
+| 12:00 - 12:30 pm    | [Introduction to Git; Forking and cloning a repository](https://docs.google.com/presentation/d/1eiGZA4PYBKJx5HDCo3UDOAB7q415gg96TehxilCHwlA/edit?usp=sharing) |
+| 12:30 - 1:30 pm     | _Lunch_                                                                                                                                                       |
+| 1:30 - 2:15 pm      | [Shell scripting](https://docs.google.com/presentation/d/1SDUyYVNgvDDRodVqmDQPVQ5wnjQesWfBTg0EAcdbcSo/edit?usp=sharing)                                       |
+| 2:15 - 2:45 pm      | [The Git stage/commit/push workflow](https://docs.google.com/presentation/d/1_YckNhAkp_82PKR6PGS5SdaKDgoueYVTXaPi5pQV9ik/edit?usp=sharing)                    |
+| 2:45 - 3:00 pm      | _Coffee break_                                                                                                                                                |
```
Comment (Jen-OMalley) [inline docs/workshop-schedule.md:28]: Just a note that they make me bring the coffee back no later than 3. But I think this should be fine. I'll just make people aware!

File: docs/workshop-schedule.md (hunk 1 above)
Comment (Jen-OMalley) [reply docs/workshop-schedule.md:33]: Agreed about removing. We may even just head straight over from the workshop!

**Review by Jen-OMalley — APPROVED** [review]
LGTM! Just added a couple small logistics comments.

**Comment by sjspielman** [comment]
@jashapiro if you end up re-reviewing/approving this while I'm out, feel free to merge!

**Review by jashapiro — APPROVED** [review]
LGTM

**Pull request merged**